Sources/BraintreeVenmo/BTVenmoDriver.m

```diff
@@ -131,48 +132,65 @@ - (void)tokenizeVenmoAccountWithVenmoRequest:(BTVenmoRequest *)venmoRequest comp
-            completionBlock(nil, error);
-            return;
+        if (venmoRequest.paymentMethodUsage != BTVenmoPaymentMethodUsageUnspecified) {
+            NSDictionary *params = @{
+                @"query": @"mutation CreateVenmoPaymentContext($input: CreateVenmoPaymentContextInput!) { createVenmoPaymentContext(input: $input) { venmoPaymentContext { id } } }",
+                @"variables": @{
+                        @"input": @{
+                                @"paymentMethodUsage": venmoRequest.paymentMethodUsageAsString,
+                                @"merchantProfileId": merchantProfileID,
+                                @"customerClient": @"MOBILE_APP",
+                                @"intent": @"CONTINUE"
+                        }
+                }
+            };
+
+            [self.apiClient POST:@"" parameters:params httpType:BTAPIClientHTTPTypeGraphQLAPI completion:^(BTJSON *body, __unused NSHTTPURLResponse *response, NSError *err) {
+                if (err) {
+                    NSError *error = [NSError errorWithDomain:BTVenmoDriverErrorDomain
+                                                         code:BTVenmoDriverErrorTypeInvalidRequestURL
+                                                     userInfo:@{NSLocalizedDescriptionKey: @"Failed to fetch a Venmo paymentContextID while constructing the requestURL."}];
+                    completionBlock(nil, error);
+                    return;
+                }
+
+                NSString *paymentContextID = [body[@"data"][@"createVenmoPaymentContext"][@"venmoPaymentContext"][@"id"] asString];
+                if (paymentContextID == nil) {
+                    NSError *error = [NSError errorWithDomain:BTVenmoDriverErrorDomain
+                                                         code:BTVenmoDriverErrorTypeInvalidRequestURL
+                                                     userInfo:@{NSLocalizedDescriptionKey: @"Failed to parse a Venmo paymentContextID while constructing the requestURL. Please contact support."}];
+                    completionBlock(nil, error);
+                    return;
+                }
+
+                NSURL *appSwitchURL = [BTVenmoAppSwitchRequestURL appSwitchURLForMerchantID:merchantProfileID
+                                                                                accessToken:configuration.venmoAccessToken
+                                                                            returnURLScheme:self.returnURLScheme
+                                                                          bundleDisplayName:bundleDisplayName
+                                                                                environment:configuration.venmoEnvironment
+                                                                           paymentContextID:paymentContextID
+                                                                                   metadata:self.apiClient.metadata];
```

Comment:
This may not be a huge cause for concern, but I wonder what implications a strong reference can carry here.